core/src/main/java/io/ultrabrew/metrics/data/ConcurrentMonoidIntTable.java

```diff
@@ -4,14 +4,15 @@
 
 package io.ultrabrew.metrics.data;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import sun.misc.Unsafe;
```

Comment:
Can you undo the non-semantic style changes. If we want to change the style, that should be a separate discussion - let's try to keep the PRs to the actual changes for now.